packages/common/src/hardhat/tasks/xchainSetup.ts

```diff
@@ -1,11 +1,33 @@
 import { Deployment } from "hardhat-deploy/types";
+import { interfaceName } from "../../Constants";
 import { task } from "hardhat/config";
 import { Contract } from "web3-eth-contract";
 import { CombinedHRE } from "./types";
 import Web3 from "web3";
 const { utf8ToHex, toBN } = Web3.utils;
 const assert = require("assert");
 
+// Note: Make sure that the following chain names are all lower case.
+const L2_CHAIN_NAMES: { [key: string]: number } = {
+  arbitrum: 42161,
+  optimism: 10,
+  boba: 288,
+  polygon: 137,
```

Comment:
nit: why do we include polygon here if it isn't currently supported by these scripts?

The double check in the verification script (one that includes polygon and one that doesn't) is a little confusing. Should we not just drop polygon (for now) and use this list as the universal source of truth?

Reply:
as unrelated nit I think this should be placed in the constants file somewhere. we have this mapping in a few places but having a canonical `chainName->chainId` mapping would be good.

Reply:
- removed polygon and the duplicate assert that the chain name is in the L2_PUBLIC_KEYS array
- I ended up using the `PublicNetworks` file already in `common` instead of redefining this chainName --> chain ID mapping